pkg/plugins/golang/v4/scaffolds/internal/templates/makefile.go

```diff
@@ -123,9 +123,9 @@ manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and Cust
 .PHONY: generate
 generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
 	{{ if .BoilerplatePath -}}
-	"$(CONTROLLER_GEN)" object:headerFile={{printf "%q" .BoilerplatePath}} paths="./..."
+	"$(CONTROLLER_GEN)" applyconfiguration:headerFile="hack/boilerplate.go.txt" object:headerFile={{printf "%q" .BoilerplatePath}} paths="./..."
 	{{- else -}}
-	"$(CONTROLLER_GEN)" object paths="./..."
+	"$(CONTROLLER_GEN)" applyconfiguration object paths="./..."
```

Comment:
So, here we should not do this change as well.
We should not change the default scaffold to be applyconfiguration

That would mean all would only work with. 

So, the first question is: 

In a project, can I have both?

Apis created with applyconfiguration and others not?
OR if we use applyconfiguration all APIs will need to generate with it?